site/en/blog/early-hints/index.md

```diff
@@ -50,13 +51,21 @@ In some cases, the performance improvement to the [Largest Contentful Paint](htt
+Now that you have this prioritized list of landing pages, the next step consists of identifying which origins or sub-resources would be good candidates for [preconnect](https://web.dev/uses-rel-preconnect) or [preload](https://web.dev/preload-critical-assets/) hints, as a first approximation. Typically, those would be origins and sub-resources that contribute the most to key user metrics such as [Largest Contentful Paint](https://web.dev/lcp/), or [First Contentful Paint](https://web.dev/first-contentful-paint/). More concretely, look for render-blocking sub-resources such as synchronous JavaScript, stylesheets, or even web fonts. Similarly, look for origins that host sub-resources that contribute a lot to key user metrics. Note: if your main resources are already using `<link rel=preconnect>` or `<link rel=preload>`, you may consider these origins or resources among the candidates for Early Hints. See [this article](https://web.dev/lcp/#how-to-improve-lcp) for more details.
 
-While this represents a decent starting point, it's not necessarily sufficient. The second step consists of minimizing the risk of using Early Hints on resources or origins that might be obsolete, or no longer used by the main resource. For instance, resources that are frequently updated and versioned (for example,  `example.com/css/main.fa231e9c.css`) may not be the best choice.  Note that this concern isn't specific to Early Hints, it applies to any link `rel=preload` or `rel=preconnect` wherever they might be present. This is the sort of detail that's best dealt with automation or templating (for example, a manual process is more likely to lead to mismatched hash or version urls between `link rel=preload` and the actual HTML tag using the resource).
+{% Aside 'important' %}
+What you decide to `preconnect` or `preload` as Early Hints may differ from what you have traditionally used these directives for in your HTML.
+
+When using these in HTML you typically want to `preconnect` or `preload` resources that the [Preload Scanner](https://web.dev/preload-scanner/) will not discover in the HTML—for example, fonts, or background images that would otherwise be discovered late. For Early Hints you will not have the HTML and so want to `preconnect` to critical domains or `preload` critical resources that perhaps _would_ be discovered early in the HTML—for example, preloading `main.css` or `app.js`.
```

Comment:
```suggestion
When using these in HTML you typically want to `preconnect` or `preload` resources that the [Preload Scanner](https://web.dev/preload-scanner/) will not discover in the HTML—for example, fonts, or background images that would otherwise be discovered late. For Early Hints you will not have the HTML and so you may want to instead `preconnect` to critical domains or `preload` critical resources that perhaps _would_ otherwise be discovered early in the HTML—for example, preloading `main.css` or `app.js`.
```